flag_preflight.py

```diff
@@ -0,0 +1,65 @@
+"""
+Naive warp:
```

Comment:
this docstring doesn't match the content or the name "flag_preflight.py" of the module anymore

Reply:
Oops will fix.